src/community/release/ext-json-ld.xml

```diff
@@ -0,0 +1,16 @@
+<assembly>
+  <id>json-ld-plugin</id>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <fileSets>
+    <fileSet>
+      <directory>release/target/dependency</directory>
+      <outputDirectory></outputDirectory>
+      <includes>	
+        <include>gs-json-ld-*.jar</include>
```

Comment:
It's probably working just fine, but I'll ask anyways... did you try it out with a GeoServer base installation, no other plugin besides this one, just dropping the contents of the zip into GeoServer, and verified the server starts and the plugin actually works? Just to be sure, I make this mistake (of missing some the necessary jars) a lot :-D

Reply:
checked it works :)